apps/src/lib/kits/maker/boards/microBit/MicroBitBoard.js

```diff
@@ -242,3 +268,5 @@ export default class MicroBitBoard extends EventEmitter {
     );
   }
 }
+
+const delay = t => new Promise(resolve => setTimeout(resolve, t));
```

Comment:
This seems like a super useful helper function! I wonder if we can have a more descriptive title here, like `delayPromise` or `promiseDelay` or add a little descriptive comment sentence.

Reply:
This comment and other nits have been addressed. PTAL @epeach  :)